nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/JoltTransformJSON.java

```diff
@@ -95,6 +99,24 @@
             .required(false)
             .build();
 
+    public static final PropertyDescriptor CUSTOM_CLASS = new PropertyDescriptor.Builder()
+            .name("jolt-custom-class")
+            .displayName("Custom Transformation Class Name")
+            .description("Fully Qualified Class Name for Custom Transformation  Module Directory should be specified")
+            .required(false)
+            .expressionLanguageSupported(false)
```

Comment:
Any reason why EL is not supported? I can't think of a great use case for using EL here, but unless there's a good reason not to, it adds to the flexibility of the system (and could eventually benefit from the Variable Registry right?)